spacepy/igrf.py

```diff
@@ -131,6 +131,12 @@ class IGRF():
 
         Dipole moments (`dict`).
     """
+    dipole = {}
```

Comment:
If we're doing this here, would it make sense to change how the dicts are populated later? Or at the very least remove line 223 (which just makes the dict again)

Reply:
This line makes a class member. Line 223 makes an instance member (which then masks this class member). If line 223 were left out, every instance of the class would share the same `dipole` dict.

It would be possible to make the class members `None`, but making them a dict helps document that this is, well, a dict.

Reply:
Ah, yes, good catch. I still dislike having the class member floating just for the sake of docs, but point taken. And if we have to have this there I'd rather it was a dict so the type is described properly.